go/kernel/linux/eventfd_linux.go

```diff
@@ -9,7 +9,6 @@ import (
 func (k *LinuxKernel) Eventfd2(initval, flags uint) uint64 {
 	var a3 uintptr
 	r1, _, err := syscall.Syscall(syscall.SYS_EVENTFD2, uintptr(&initval), uintptr(&flags), a3)
-	fmt.Println(r1)
 	if err != 0 {
 		posix.Errno(err)
```

Comment:
I think `err` should actually be `errn` or `errno` and it should be returned without `posix.Errno()` as per https://github.com/lunixbochs/usercorn/pull/239#issuecomment-298914412

Like `return errn` here.